src/main/webapp/dataverse.xhtml

```diff
@@ -287,6 +287,7 @@
                                                         <p:ajax update="@widgetVar(optionBlock)" />
                                                     </input>
                                                     <input type="checkbox" jsf:itemValue="#{mdb}" id="#{mdb.idString}" jsf:value="#{mdb.selected}"
+                                                           jsf:disabled="#{DataversePage.editInputLevel == true}"
```

Comment:
Does this still work without the `jsf:` passthrough? I would imagine it would. The other sibling basic HTML input's above it do not have it on their disabled attributes.

Reply:
It doesn't work without the jsf: because if the disabled tag is there even if it contains "false" it disables the component. The others on lines 284 and 286 have disabled="disabled" and the real decision work is done on the jsf:rendered.